.github/workflows/build.yml

```diff
@@ -14,12 +14,39 @@ on:
 
 env:
   BUILD_VERSION: "2.6.1"
+  CONFIGURATION: "Release"
+  GA_USE_GITHUB_ENV: "true"
+  USE_MSSQL_DEFAULT_CONN_STRING: "FALSE"
+  USE_POSTGRES_DEFAULT_CONN_STRING: "TRUE"
+  SQLSERVER_SERVER: "localhost"
+  SQLSERVER_SA_PASS: ${{ secrets.MSSQL_SA_PWD }}
+  SQLSERVER_DATABASE_DS2: "AnalyticsMiddleTier_Testing_Ds2"
+  SQLSERVER_DATABASE_DS31: "AnalyticsMiddleTier_Testing_Ds31"
+  SQLSERVER_DATABASE_DS32: "AnalyticsMiddleTier_Testing_Ds32"
+  SQLSERVER_INTEGRATED_SECURITY: "false"
+  SQLSERVER_USER: ${{ secrets.MSSQL_USER }}
+  SQLSERVER_PASS: ${{ secrets.MSSQL_PWD }}
+  POSTGRES_HOST: "localhost"
+  POSTGRES_DATABASE: "edfi_ods_tests"
+  POSTGRES_PORT: "5432"
+  POSTGRES_USER: ${{ secrets.POSTGRESQL_USER }}
+  POSTGRES_PASS: ${{ secrets.POSTGRESQL_PWD }}
+  POSTGRES_POOLING: "false"
 
 jobs:
   build:
-    runs-on: windows-latest
+    #runs-on: Windows-latest
+    runs-on: Ubuntu-latest
 
     steps:
+      - name: Start windows container for SQLServer 2017 Express
```

Comment:
This is actually a Linux-based container.

Reply:
Yes, I changed the script to use Ubuntu instead of Windows.